rental.rb

```diff
@@ -0,0 +1,16 @@
+# Public: Rental used to create a rental object.
+#
+# All methods are class methods and should be called on the Book class.
+class Rental
+  attr_accessor :date
```

Comment:
- [ ] Also, kindly ensure that _Getters_ and _Setters_ for @book and @person instance variables are present:
```ruby
  attr_accessor :date, :book, :person
```